setup.py

```diff
@@ -62,10 +64,15 @@ def read(*rnames):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Topic :: Internet :: WWW/HTTP',
-        'Framework :: Zope :: 3'],
+    ],
     zip_safe=False,
-    entry_points={'distutils.commands': 'ftest = zope.testrunner.eggsupport:ftest'}
-    )
+    entry_points={
+        'distutils.commands': 'ftest = zope.testrunner.eggsupport:ftest'
+    }
```

Comment:
TBH I think this can be dropped entirely.

Reply:
Done.